docs/setup.md

```diff
@@ -1,53 +1,62 @@
 # Setup
 
-In order to generate Hootenanny RPMs, you'll need RPM tools, Docker, Vagrant, and `make`.
-Fortunately, scripts are included to automate the setup process for Linux users.
-The scripts assume:
+In order to generate Hootenanny RPMs, you'll need RPM tools, Docker, Vagrant,
```

Comment:
Nit-picky here and I may be wrong, but the Hootenanny RPMs can be generated with RPM tools, Docker and make alone, correct?

Reply:
Good point.  I'll clarify that it's preferred to use Vagrant and `make`, and point to the shell documentation (which needs to exist first).

Reply:
Fixed in 34d9ddf.